modules/libcom/src/osi/os/WIN32/epicsAtomicOSD.h

```diff
@@ -19,27 +19,50 @@
 
 #define EPICS_ATOMIC_OS_NAME "WIN32"
 
-#ifdef VC_EXTRALEAN
-#   define VC_EXTRALEAN_DETECTED_epicsAtomicOSD_h
-#else
-#   define VC_EXTRALEAN
-#endif
+/* Disable extra declarations that we don't need here (i.e. winsock1, rpc, etc.) */
+#pragma push_macro("WIN32_LEAN_AND_MEAN")
+#undef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
 
-#ifdef STRICT
-#   define STRICT_DETECTED_epicsAtomicOSD_h
-#else
-#   define STRICT
-#endif
+#pragma push_macro("STRICT")
+#undef STRICT
+#define STRICT
```

Comment:
To be sure of `STRICT` I think we also need to push and undef `NO_STRICT`  https://learn.microsoft.com/en-us/windows/win32/winprog/disabling-strict

Reply:
Actually, on reflection if somebody has explicitly defined `NO_STRICT` then we should leave as is for include - so ignore this suggestion